build-logic/conventions/src/main/kotlin/com/squareup/anvil/conventions/BasePlugin.kt

```diff
@@ -135,6 +137,12 @@ abstract class BasePlugin : Plugin<Project> {
         target.layout.buildDirectory.set(target.file("build/included-build"))
       }
     }
+
+    // Set the kase working directory ('<build directory>/kase') as a System property,
+    // so that it's in the right place for projects with relocated directories.
+    target.tasks.withType(Test::class.java).configureEach { task ->
```

Comment:
Can you add a link to https://github.com/rickbusarow/kase/blob/255db67f40d5ec83e31755bc9ce81b1a2b08cf11/kase/src/main/kotlin/com/rickbusarow/kase/files/HasWorkingDir.kt#L93-L96